setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = kuksa_client
+author = Sebastian Schildt, Naresh Nayak, Wenwen Chen
+author_email = sebastian.schildt@de.bosch.com, naresh.nayak@de.bosch.com, wenwen.chen@de.bosch.com
+description = kuksa.val python clients and SDK
```

Comment:
I think the name __has to__  to include VISS, becasue that is what is implementing

kuksa_viss

or

kuksa_viss_client

also it